### ISSUE TITLE
Connection Editor does not respect allowed classes

### ISSUE THREAD

**Issue by jensschmeink** [dlr-gtlab/gtlab-core#1147]
Calculator A has object link property with GtdRotorBladeRow as only allowed class. Calculator B has object link property with GtdStatorBladeRow as only allowed class. The Connection Editor allows these two properties to be connected.
It would be nice if the Connection Editor would not allow this. Else, there is additional workload for the process developer in order to handle errors.

Originally posted by O.K. in bug tracking group

**Comment by jensschmeink** [comment]
Familiar topic: add warnings if connections between double values are established which does not use the same unit

**Comment by jensschmeink** [comment]
@OliverKunc I was thinking about this issue: It is not that easy to find a general solution as object links have a list of allowed classes.    
Example:  
In Calc A the property can be set to a compressor or an turbine.  
In Calc B the property can be set to a turbine or a bladerow.  

What would be the behavior you expect?

**Comment by OliverKunc** [comment]
Good point. I'd expect the connection to be allowed. My default idea is that obvious errors should be prevented. In your example it is not obvious, so it is the user's responsibility.
